Configuration/Generator/python/TenE_E_0_200_pythia8_cfi.py

```diff
@@ -0,0 +1,22 @@
+import FWCore.ParameterSet.Config as cms
+
+# Modified from Configuration/Generator/python/TenMuE_0_200_pythia8_cfi.py
+generator = cms.EDFilter("Pythia8EGun",
```

Comment:
I don't know why you need the string in general, but I think that you can choose it. See for example here:
https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideGenFilters

Anyhow, as one either produces ten muons or ten electrons, I don't see a big overlap problem here.